build/make-distribution.sh

```diff
@@ -262,7 +256,8 @@ function sbt_build_service {
 
   echo "Celeborn $VERSION$GITREVSTRING" > "$DIST_DIR/RELEASE"
   echo "Build flags: $@" >> "$DIST_DIR/RELEASE"
-  if [[ "$HADOOP_AWS_ENABLED" == "true" ]]; then
+
+  if [[ $@ == *"hadoop-aws"* ]]; then
     export SBT_MAVEN_PROFILES="hadoop-aws"
```

Comment:
There is no need to export this variable. Just setting this variable should be enough.

Reply:
Done. Thanks